src/plugins/ganttChart/dateCalculator.js

```diff
@@ -26,6 +26,14 @@ class DateCalculator {
      * The current `allowSplitWeeks` option state.
      */
     this.allowSplitWeeks = allowSplitWeeks === void 0 ? true : allowSplitWeeks;
+    /**
+     * The current `hideDaysBeforeFullWeeks` option state.
+     */
+    this.hideDaysBeforeFullWeeks = hideDaysBeforeFullWeeks === void 0 ? false : hideDaysBeforeFullWeeks;
```

Comment:
Please use the default function parameters above for `hideDaysBeforeFullWeeks`, `hideDaysAfterFullWeeks` and `allowSplitWeeks`.